marbles/core/tests/test_marbles.py

```diff
@@ -480,6 +493,14 @@ def test_verify_annotation_locals(self):
         e = ar.exception
         self.assertEqual(e.note.strip(), "some note about 'bar'")
 
+    def test_verify_multiline_locals(self):
```

Comment:
I don't love this test method name; how about `test_mulitline_locals_indentation`?